ajax/libs/pulltorefreshjs/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "pulltorefreshjs",
+  "filename": "pulltorefresh.min.js",
+  "version": "0.1.0",
+  "description": "Pull To Refresh",
+  "keywords": [
+    "pulltorefresh",
+    "webapp",
+    "javascript"
```

Comment:
@kennynaoh do you made the keywords yourself? I don't think "javascript" is good to be a keyword.